src/components/dialogs/equipment-deletion-dialog.js

```diff
@@ -71,19 +111,29 @@ const EquipmentDeletionDialog = ({
     const inputForm = useInputForm();
 
     const [equipmentType, setEquipmentType] = useState(
-        editData?.equipmentType ?? defaultEquipmentType
+        equipmentTypes[editData?.equipmentType] ?? defaultEquipmentType
     );
 
+    const [equipmentsFound, setEquipmentsFound] = useState([]);
+
     const [errors, setErrors] = useState(new Map());
 
-    const [searchMatchingEquipments, equipmentsFound] =
-        useSearchMatchingEquipments(
-            studyUuid,
-            currentNodeUuid,
-            true,
-            equipmentType,
-            makeItems
-        );
+    useEffect(() => {
+        setEquipmentsFound([]);
```

Comment:
We can avoid this set ?

Reply:
If we remove this set, a glitch can appear when displaying equipments list in the autocomplete (when moving from one equipment type to another)